README.md

```diff
@@ -56,6 +56,13 @@ current_version = Stanford::StorageServices.current_version('oo000oo0000') # whe
 expect(current_version).to be_an_instance_of Integer
 ```
 
+#### Get Size of Moab Object
+
+```ruby
+object_size_in_bytes = Stanford::StorageServices.object_size('oo000oo0000') # where oo000oo0000 is the druid
+expect(object_size_in_bytes).to be_an_instance_of Integer
```

Comment:
I think the implementation isn't stanford specific, so perhaps the README changes should be in both generic and stanford specific places??

Reply:
agreed. just updated it!